apps/meet-app/backend/modules/calendar/utils.bal

```diff
@@ -0,0 +1,145 @@
+    string updatedDescription = updatedDisclaimer + separator + createCalendarEventRequest.description;
+
+    // Format the event payload as required by the Google Calendar API.
+    CreateCalendarEventPayload calendarEventPayload = {
+        summary: createCalendarEventRequest.title,
+        description: updatedDescription,
+        'start: {
+            dateTime: createCalendarEventRequest.startTime,
+            timeZone: createCalendarEventRequest.timeZone
+        },
+        end: {
+            dateTime: createCalendarEventRequest.endTime,
+            timeZone: createCalendarEventRequest.timeZone
+        },
+        attendees: [
+            ...createCalendarEventRequest.wso2Participants.map((email) => ({email: email.trim()})),
+            ...createCalendarEventRequest.externalParticipants.map((email) => ({email: email.trim()}))
+        ],
+        guestsCanModify: true,
+        conferenceData: {
+            createRequest: {
+                requestId: uuid:createType4AsString(),
+                conferenceSolutionKey: {
+                    'type: "hangoutsMeet"
+                }
+            }
+        }
+    };
+
+    // Create the payload for the API request.
+    http:Request req = new;
+    json calendarEventPayloadJson = calendarEventPayload.toJson();
+    req.setPayload(calendarEventPayloadJson);
+
+    // Make the API call to create the event.
+    http:Response|error response = calendarClient->post(string `/events/${calendarId}?sendUpdates=all`, req);
+    if response is error {
+        return error("Failed to create the calendar event.");
+    }
+
+    // Check if the event was created successfully.
+    if response.statusCode == 201 {
+        json responseJson = check response.getJsonPayload();
+        CreateCalendarEventResponse createCalendarEventResponse = check responseJson
+        .cloneWithType(CreateCalendarEventResponse);
+        return createCalendarEventResponse;
+    }
+
+    return error("Failed to create the calendar event.");
+}
+
+# Delete an event from the calendar.
+#
+# + eventId - Event Id
+# + return - JSON response if successful, else an error
+public isolated function deleteCalendarEvent(string eventId) returns DeleteCalendarEventResponse|error {
+
+    // Make the API call to delete the event.
+    http:Response|error response = calendarClient->delete(string `/events/${calendarId}/${eventId}`);
+    if response is error {
+        return error("Failed to delete the calendar event.");
+    }
+
+    // Check if the event was deleted successfully.
+    if response.statusCode == 200 {
+        json responseJson = check response.getJsonPayload();
+        DeleteCalendarEventResponse deleteCalendarEventResponse = check responseJson
+        .cloneWithType(DeleteCalendarEventResponse);
+        return deleteCalendarEventResponse;
+    }
+
+    return error("Failed to delete the calendar event.");
```

Comment:
Here also, you need to return the other http responses other than of HTTP 200. at least create error with the actually response message.